tomviz/CentralWidget.cxx

```diff
@@ -53,6 +53,10 @@
 # include "dax/ModuleStreamingContour.h"
 #endif
 
+Q_DECLARE_METATYPE(vtkSmartPointer<vtkImageData>)
```

Comment:
What is the motivation to pass vtkSmartPointer around rather than the pointer? I can see why we would do this with a C++ shared pointer object, but not sure I see the motivation here due to the internal reference counting in vtkObject.